### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by pmwtsn**
This supercedes PR #649.

### CODE REVIEW

**Comment by markhibberd** [comment]
@pmwtsn For reference: http://ivory.readthedocs.org/en/latest/install/

**Comment by pmwtsn** [comment]
@markhibberd - thanks Mark. I'll fix the title.

Also, @blever noted that the curl command towards the end points to an old version... any idea what that should be now?

**Comment by markhibberd** [comment]
@pmwtsn I will get it updated.

**Comment by pmwtsn** [comment]
@markhibberd cheers

**Comment by charleso** [comment]
:+1:

File: guide/install.md [hunk 1]
```diff
@@ -1,4 +1,71 @@
-Install
-=======
+---
+title: Install Ivory
```
Comment (charleso) [inline guide/install.md]: @pmwtsn FYI This should just be the normal markdown title (with the equal signs). We're not using hackyll any more, and this is now displayed incorrectly. Not a big deal, but just something to watch out for as you go.

**Pull request merged**